### DISCUSSION TITLE
document bunjs hosting

### DISCUSSION THREAD

**Discussion by elycheikhsmail** [domcloud/domcloud#55] [General]
I hope doc about hosting bunjs based app using domcloud. bunjs us relatively new, for this raison I'd like domcloud to be one the first plateforme that allow hosting bunjs app. in one side it will made is learning and experemting this new js runtime, and give opportunity for more developer to discever domcloud. it will be win-win situation.

**Comment by willnode** [comment]
The documentation [has been updated](https://domcloud.co/docs/deployment/bun).

**Reply by elycheikhsmail** [reply]
Please replace "bun is recently popular programing language " by " bun is recently popular js runtime ..." as you know the language is javascript. I know sometime we can confuse runtime and language when we are sure no confusion will happened. In bun discord some people was confused.

**Reply by willnode** [reply]
done updated
